opentech/settings/base.py

```diff
@@ -674,3 +674,35 @@
 PROJECTS_ENABLED = False
 if env.get('PROJECTS_ENABLED', 'false').lower().strip() == 'true':
     PROJECTS_ENABLED = True
+
+
+# Debug Toolbar
+# https://django-debug-toolbar.readthedocs.io/en/latest/index.html
+if DEBUG:
+    INSTALLED_APPS.append('debug_toolbar')
+    MIDDLEWARE = [
+        'debug_toolbar.middleware.DebugToolbarMiddleware',
+        *MIDDLEWARE,
```

Comment:
I notice you changed the pattern here. Just a preference or does it have benefits? Still learning Python so curious.

Reply:
It avoids mutating the `MIDDLEWARE` setting.  Mutation is typically considered non-ideal because it often has side effects.

I've just noticed I didn't used it for `INSTALLED_APPS`, I'll update that line to have the same pattern.